packages/wdio-applitools-service/src/@types/@applitools/eyes-webdriverio.d.ts

```diff
@@ -0,0 +1,27 @@
+/// <reference types="@applitools/visual-grid-client"/>
+/// <reference types="../applitools-service"/>
```

Comment:
Are these needed?

Reply:
I'd like to avoid duplicating `ProxySettings` - https://github.com/webdriverio/webdriverio/pull/5981/files#diff-b360aff2f1d276bb6ea28b5b1ab3888db08d0a2be1da0dd51405e0bf26e34987R8-R14. Both `eyes-webdriverio.d.ts` and `applitools-service.d.ts` uses `ProxySettings` interface.